packages/basic-crawler/src/internals/basic-crawler.ts

```diff
@@ -677,6 +692,7 @@ export class BasicCrawler<Context extends CrawlingContext = BasicCrawlingContext
 
         try {
             this.log.info('Starting the crawl');
+            await periodicLogger.log();
```

Comment:
@barjin this also fixes the issue I was mentioning with status message being fired too early